guide/how-it-works/payment-request-formats.md

```diff
@@ -19,119 +19,235 @@ images:
 
-<span style="word-break: break-word;">LNURL1DP68GURN8GHJ7MRWW4EXCTT5DAHKCCN00QHXGET8WFJK2UM0VEAX2UN09E3K7MF0W5LHGCT884KX7EMFDCNXKVFAVGURWWT9XYCNYE35XUMRXCEHVY6NGDEHVSUNWWPEV5UN2VMY8Q6RGDF4XVEXYE33V56NSETRVVMRGEFEVCMRWENPXCUXYEP48PNX2EQPG3G5S</span>
+A downside to payment codes is they require an additional on-chain transaction each time two users want to connect and use them.
 
-{% include fact/close.html %}
+</div>
 
-{% include fact/open.html color="blue" label="LNURL-Channel" icon="triangle" %}
+### Silent payment
 
-<span style="word-break: break-word;">LNURL1DP68GURN8GHJ7MRWW4EXCTT5DAHKCCN00QHXGET8WFJK2UM0VEAX2UN09E3K7MF0W5LHZ0F4VFJK2CEHX5CR2DT9XP3XZWP4V5EN2VF489SKYCFJX5UR2VTYV4SKYWRZX93RSEPJV9SKGVRRXYER2WT9XEJNZEPKX4NRGDEEXS6R2XNW3D6</span>
+Silent payments is a [newly proposed format](https://gist.github.com/RubenSomsen/c43b79517e7cb701ebf77eec6dbb46b8?permalink_comment_id=4113680) that allows two users to privately make an on-chain payment between them. This is similar to payment codes however does not need to establish a relationship between users limiting their on-chain footprint.
+
+The trade-off that silent payments require is receivers of silent payments require extra bandwidth to scan the bitcoin UTXO set to find payments made to them. This could be problematic on mobile and other less powerful devices.
+
+## Bitcoin uniform resource identifiers (URIs)
+
+A [Uniform Resource Identifier](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) (URI) is a web technology uses unique prefixes to identify anything. This can be real-world objects, websites, information, etc.
+
+Bitcoin leverages this technology when dealing with payment requests. The technical details about how this is done is detailed in [BIP 21](https://bips.xyz/21). But essentially bitcoin payment requests use a `bitcoin:` URI scheme to identify and help applications manage bitcoin payment requests.
+
+This scheme is extensible and allows for the addition of other useful meta data that give more context to a payment request. This includes things like an amount, labels, messages and other useful data. It also allows more than one type of payment request format to be included in a payment request. We explore this in the next section on unified payment requests.
+
+## Unified payment requests
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/payment-request-formats/unified.jpg"
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/unified@2x.jpg"
+   alt-text = "A QR code of an offer"
+   width = 400
+   height = 500
+   layout = "float-right-desktop"
+%}
 
-{% include fact/close.html %}
+A unified payment request combines one or more of the formats listed above. This makes it easier to request a payment when you are unsure what formats the sender supports.
```

Comment:
Isn't the unified payment request stuff we've been hyped about just focused on basic on-chain addresses plus a lightning invoice? Or are there also implementations of on-chain address with a LNURL-Pay code, or other combos?